packages/plugins/io-center/src/ui/components/canvas/canvas-area.svelte

```diff
@@ -26,16 +26,6 @@
 
 	onMount(() => {
 		window.addEventListener("resize", redrawConnections);
```

Comment:
it probably does not belong to this PR but we shouldn't add event listeners manually.
in this case we should use: https://svelte.dev/docs/svelte/svelte-window